doc/rst/language/spec/conversions.rst

```diff
@@ -768,18 +795,28 @@ expression or a type expression.
 Explicit Range Conversions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-An expression of a range type with ``strides=strideKind.any``
-can be explicitly converted to a range type with ``strides=strideKind.one``,
-changing the stride to 1 in the process.
+An expression of a range type can be explicitly converted to another
+range type with the same ``bounds`` parameter. Upon such conversion,
+each non-infinite bound of the source is explicitly converted
+to the target's ``idxType``. The explicit conversion for ranges
+is not allowed when the explicit conversion between their ``idxTypes``
+is not allowed.
+
+The explicit conversion results in an error when the ``stride`` value
+of the source is not legal for the target type. This may be the case
```

Comment:
runtime or compile-time? I'd expect run-time, might be good to mention either way here.

Reply:
I am leaving this unspecified because in the current implementation it can be either.